Source/Core/Core/NetPlayClient.cpp

```diff
@@ -427,6 +428,13 @@ unsigned int NetPlayClient::OnData(sf::Packet& packet)
           g_NetPlaySettings.m_CPUcore = PowerPC::CPUCore::CachedInterpreter;
       }
 
+      {
+        bool follow_branch;
+        packet >> follow_branch;
+
+        Config::SetCurrent(Config::MAIN_JIT_FOLLOW_BRANCH, follow_branch);
```

Comment:
A field should be added to NetSettings for this, and the config set in the NetPlayConfigLoader, not here.